packages/terra-form-checkbox/docs/README.md

```diff
@@ -38,6 +38,12 @@ import Checkbox from 'terra-form-checkbox';
 <Checkbox id="checkbox" labelText="Default Checkbox" />
 ```
 
+## Jest Tests:
+* Add a mock for `window.matchMedia` in case Jest reports an error saying `TypeError: window.matchMedia is not a function`.
```

Comment:
Perhaps this section could be edited to say (or similar):
## Jest Tests
* This component requires `window.matchMedia` to be mocked in Jest tests. To do so add the following to the top of your test file(s) that use this component:
```
window.matchMedia = () => ({ matches: true });
```

````
## Jest Tests
* This component requires `window.matchMedia` to be mocked in Jest tests. To do so add the following to the top of your test file(s) that use this component:
```
window.matchMedia = () => ({ matches: true });
```
````